spec/services/big_blue_button_api_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'bigbluebutton_api'
+
+describe BigBlueButtonApi, type: :service do
+  let(:bbb_server) { described_class.new }
+  let(:bbb_api) { bbb_server.bbb_server }
```

Comment:
Does this need to be accessible for the entire service? if not, I would just move the logic down to the specific test itself (see below)